src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -45,10 +46,15 @@ import { sidebarStyles } from '../Sidebar.styles';
 import { isJsonString } from './sample-query-utils';
 
 export class SampleQueries extends Component<ISampleQueriesProps, any> {
+  resetCollapse: boolean;
```

Comment:
better to have explicit modifier specified for every class member such as `private _resetCollapse: boolean;` also it's a convention to add `_` for private members

Reply:
Thanks!!